docs/01-app/02-guides/prefetching.mdx

```diff
@@ -0,0 +1,204 @@
+
+import { useRouter } from 'next/navigation'
+import { useEffect } from 'react'
+
+function ManualPrefetchLink({
+  href,
+  children,
+}: {
+  href: string
+  children: React.ReactNode
+}) {
+  const router = useRouter()
+
+  useEffect(() => {
+    let cancelled = false
+    const poll = () => {
+      if (!cancelled) router.prefetch(href, { onInvalidate: poll })
+    }
+    poll()
+    return () => {
+      cancelled = true
+    }
+  }, [href, router])
+
+  return <a href={href}>{children}</a>
+}
+```
+
+`onInvalidate` is invoked when Next.js suspects cached data is stale, allowing you to refresh the prefetch.
+
+> **Good to know:** Using an `a` tag will cause a full page navigation to the destination route, you can use `onClick` to prevent...
+
+## Disabled prefetch
+
+You can fully disable prefetching for certain routes for more fine-grained control over resource consumption.
+
+```tsx
+'use client'
+
+import Link, { LinkProps } from 'next/link'
+
+function NoPrefetchLink({
+  prefetch,
+  ...rest
+}: LinkProps & { children: React.ReactNode }) {
+  return <Link {...rest} prefetch={false} />
+}
+```
+
+For example, you may still want to have consistent usage of `<Link>` in your application, but links if your footer might not need to be prefetched when entering the viewport.
+
+## Prefetching optimizations
+
+### Layout deduplication
+
+Shared layouts download once per session, even if several prefetches request the same component tree. This reduces network traffic and speeds up subsequent navigations that reuse the layout.
+
+### Prefetch scheduling
+
+Next.js maintains a small task queue, which prefetches in the following order:
+
+1. Links in the viewport
+2. Links showing user intent (hover or touch)
+3. Newer links replace older ones
+4. Links scrolled off-screen are discarded
+
+The scheduler prioritizes likely navigations while minimizing unused downloads.
+
+### Partial Prerendering (PPR)
+
+With PPR is enabled, a page is divided into a static shell and a streamed dynamic section:
+
+- The shell, which can be prefetched, streams immediately
+- Dynamic data streams when ready
+- Data invalidations (`revalidateTag`, `revalidatePath`) silently refresh associated prefetches
+
+### Router cache
+
+Prefetched React Server Component payloads are stored in memory and keyed by route segments. Navigating between sibling routes—e.g. `/dashboard/settings` → `/dashboard/analytics`—reuses parent payloads and fetches only the changed leaf.
+
+## Troubleshooting
+
+### Triggering unwanted side-effects during prefetching
+
+- Server Actions and Client Components
+
+### Preventing too many prefetches
+
+- What is the problem with prefetching too much? Unnecessary server load. Especially for routes users might never visit.
+- Hover-trigger the prefetch or use manual prefetching
+
+### Disabling prefetching on hover
+
+- How do you disable prefetch on hover? https://github.com/vercel/next.js/discussions/24437
+- Use your own Ejected Link component
```

Comment:
Historically, this was a problem in pages router, because disabling prefetch, kept the on-hover behavior. 

In this case, the root issue was "too many prefetches", that lead people to opt-into prefetch false, but, perhaps unexpectedly, still kept the on-hover issue.

In App Router, prefetch false, disables on-hover too. So that problem is solved. However, we have the opposite problem, so to speak. People who did want to disable prefetching on the viewport, but still prefetch on hover.

The proposal up above, to do a `Hover-triggered prefetch` ~ does cover the use case, so I think we can drops this third section altogether.

```suggestion
```